polaris-react/src/components/Text/Text.scss

```
@@ -164,6 +164,7 @@
 
 .break {
   overflow-wrap: break-word;
+  word-break: break-word;
```

This is the suggested migration from `word-break: break-word;`, I tested and it works!

```suggestion
  overflow-wrap: anywhere;
  word-break: normal;
```

---

Thank you!